docs/docs/plugin-authoring.md

```diff
@@ -64,3 +65,20 @@ Like all `gatsby-*` files, the code is not processed by Babel. If you want
 to use JavaScript syntax which isn't supported by your version of Node.js, you
 can place the files in a `src` subfolder and build them to the plugin folder
 root.
+
+## Publishing a plugin to the library
+
+In order to add your plugin to the [Plugin Library], you need to simply publish a package to NPM (learn how [here](https://docs.npmjs.com/getting-started/publishing-npm-packages)) with the [required files](#what-files-does-gatsby-look-for-in-a-plugin) and **include a `keywords` field** to `package.json` containing `gatsby` and `gatsby-plugin`.
```

Comment:
We try to avoid words like "simply", because to a first-timer, publishing a package to npm may not feel simple. Could you remove "simply" here?

Also, npm branding dictates that their name is all lowercase.